develop-docs/sdk/telemetry/logs.mdx

```diff
@@ -227,16 +227,10 @@ logger()->info('Adding item %s for user %s', [$item_id, $user_id], ['extra' => 1
 import io.sentry.Sentry;
 
 // example with MessageFormat based string template
```

Comment:
we can remove this comment - it's not `MessageFormat` based 😅

Reply:
```suggestion
```